crypto/types/src/types.rs

```diff
@@ -817,6 +839,52 @@ impl PublicKey {
             compressed: new_pk.to_encoded_point(true),
         }
     }
+
+    /// Generates a new random public key.
+    /// Because the corresponding private key is discarded, this might be useful only for testing purposes.
+    pub fn random() -> Self {
```

Comment:
So that it generates less code in the production, where it is not used.
```suggestion
    /// Because the corresponding private key is discarded, this might be useful only for testing purposes.
    #[inline]
    pub fn random() -> Self {
```